circle/functions

```diff
@@ -105,8 +106,22 @@ is_default_image() {
   return 0
 }
 
+get_default_distro() {
+  if is_default_distro_transition; then
+    echo "${DEFAULT_DISTRO}"
+    return 0
+  fi
+
+  echo "${DEPRECATED_DEFAULT_DISTRO}"
+  return 0
+}
+
 is_default_distro() {
-  [[ $1 == ${DEFAULT_DISTRO} ]]
+  [[ $1 == "$(get_default_distro)" ]]
+}
+
+is_default_distro_transition() {
+  [[ $DISTRIBUTIONS_LIST == *${DEPRECATED_DEFAULT_DISTRO}* ]] && [[ $DISTRIBUTIONS_LIST == *${DEFAULT_DISTRO}* ]]
```

Comment:
This will fail when we remove `debian-8`.
The logic should be if `DISTRIBUTIONS_LIST` includes `debian-9`, the default is `debian-9`, otherwise `debian-8`.